extensions/common/context.h

```diff
@@ -134,6 +134,12 @@ struct RequestInfo {
   std::string url_path;
   std::string url_host;
   std::string url_scheme;
+
+  // TCP variables.
+  int64_t tcp_connections_opened = 0;
```

Comment:
We should try to use the information that is available in upstream instead of doing the computation inside the filter. I think stream_info also records the bytes sent and received for tcp, so we probably should add a property to retrieve them: https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/common/expr/context.cc

Reply:
I tried it but value was coming out to be 0.
My hunch is that because we use http streaminfo only in wasm and not connection stream info..
thus, this value is 0..